app/main.py

```diff
@@ -1,25 +1,73 @@
 from abc import ABC
+from typing import Callable
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int,
+                 max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: int, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: str, owner: int) -> None:
+        getattr(instance, self.protected_name)
```

Comment:
The `__get__` method in the `IntegerRange` class should return the value of the attribute. Currently, it calls `getattr` but does not return the result. Please ensure you return the result of `getattr(instance, self.protected_name)`.